app/lib/lets-encrypt.server.ts

```diff
@@ -324,7 +324,7 @@ class LetsEncrypt {
 
     /**
      * Validate challenges that are in the `pending` or `invalid` state
-     * It is possible retry failed challenges
+     * It is possible to retry failed challenges 
```

Comment:
Prettier will be upset with that trailing space at the end

Reply:
yeah, used the web editor, not sure how that got there. Fixed now